iris_pipeline_project/convert.py

```diff
@@ -0,0 +1,24 @@
+from sklearn.datasets import load_iris
```

Comment:
This entire file should be wrapped in OvmsPythonModel with training done in `execute` method